wandb/util.py

```diff
@@ -855,8 +857,7 @@ def prompt_choice():
 
         if not key:
             wandb.termlog('Create an account here: {}/login?signup=true'.format(api.app_url))
-            wandb.termlog('You can find you API key in your browser here: {}/authorize'.format(api.app_url))
-            key = getpass.getpass('%s: Paste an API key from your profile and hit enter: ' % wandb.core.LOG_STRING).strip()
+            key = input_callback('%s: Paste an API key from your profile an hit enter' % wandb.core.LOG_STRING).strip()
```

Comment:
typo:
"an hit enter" -> "and hit enter"

Reply:
we tell the user to go to login?signup=true
but we also open up http://app.wandb.ai/authorize
which has a "Log in or signup to view this page."
Will this signup also pass the "signup=true"?